robust-checks.tex

```diff
@@ -1,134 +1,29 @@
 \documentclass[10pt,letterpaper]{article}
-\usepackage[top=0.85in,left=2.75in,footskip=0.75in]{geometry}
-
-% amsmath and amssymb packages, useful for mathematical formulas and symbols
-\usepackage{amsmath,amssymb}
-
-% Use adjustwidth environment to exceed column width (see example table in text)
-\usepackage{changepage}
+\usepackage[top=0.85in,left=1.0in,footskip=0.75in]{geometry}
```

Comment:
No, this is for the checklist. It doesn't need to be in PLOS format, since it's supplemental. I tweaked it until it looked good and everything fit on one page.